tomviz/LoadDataReaction.cxx

```diff
@@ -48,27 +50,38 @@ LoadDataReaction::~LoadDataReaction()
 //-----------------------------------------------------------------------------
 void LoadDataReaction::onTriggered()
 {
-  vtkNew<vtkSMParaViewPipelineController> controller;
+  loadData();
+}
 
-  // Let ParaView deal with reading the data. If we need more customization, we
-  // can show our own "File/Open" dialog and then create the appropriate reader
-  // proxies.
-  QList<pqPipelineSource*> readers = pqLoadDataReaction::loadData();
+QList<DataSource*> LoadDataReaction::loadData()
+{
+  vtkNew<vtkSMParaViewPipelineController> controller;
 
-  // Since we want to ditch the reader pipeline and just keep the raw data
-  // around. We do this magic.
-  foreach (pqPipelineSource* reader, readers)
+  QStringList filters;
+  filters << "Common file types (*.jpg *.png *.tiff *.raw *.txt)"
```

Comment:
I think we should add *.dat, *.jpeg, *.tif and *.bin up here (and they should be added below too).